sql-training/src/ex10.sql

```diff
@@ -0,0 +1,10 @@
+-- In a single query find the latest consumption count for different types of consumptions. 
+-- The fact_facebook_post_consumptions_by_type_samples is the table for reference.
+
+SELECT  f.consumption_type, f.consumptions_count , f.fact_ts

```

Comment:
facebook_post_dcs_uid | link_clicks | other_clicks | video_play |
---------------------------------+---------------+----------------+---------------+
shdfah234akjndfjndjnf345 | 3455        |   3322         |  3466


The consumption types should be in columns